src/System.Management.Automation/engine/CommandMetadata.cs

```diff
@@ -1107,6 +1111,17 @@ internal string GetEndBlock()
 ";
         }
 
+        internal string GetCleanupBlock()
+        {
+            return @"
+    try {
+        $steppablePipeline.Dispose()
+    } catch {
+        throw
+    }
```

Comment:
No idea. I just copied the existing patterns for these generated strings. All of them have a `try/catch` where the `catch` does nothing other than `throw`.

I'm all for removing it, but didn't want to touch it in case there's some weird edge case that proxy commands need this to be here for. Even then, though, I'm pretty sure the weirdness around `throw` means this should instead be `throw $_` (iirc there are some slight differences in the result, there's an old issue around it that nobody's been game to touch for years).